app/models/pregnancy.rb

```diff
@@ -67,4 +68,34 @@ def recent_calls
   def old_calls
     calls.order('created_at DESC').offset(10)
   end
+
+  def contact_made?
+    self.calls.each do |call|
+      return true if call.status == "Reached patient"
+    end
+    false
+  end
+
+  def pledge_status?(status)
+    self.pledges.each do |pledge|
+      return true if pledge[status]
+    end
+    false
+  end
+
+  def infer_status
```

Comment:
Since we're basically going to be using this in place of `status`, let's get rid of the entire `status` field and replace it with this. 

In other words -- Can we rename this method to `status`, and delete the model attribute defined above?